src/adapter/etl-adapter-google-sheet/src/Flow/ETL/Adapter/GoogleSheet/RowsNormalizer/EntryNormalizer.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\ETL\Adapter\GoogleSheet\RowsNormalizer;
+
+use function Flow\ETL\DSL\{date_interval_to_microseconds, type_json};
+use Flow\ETL\PHP\Type\Caster;
+use Flow\ETL\Row\Entry;
+
+final readonly class EntryNormalizer
+{
+    public function __construct(
+        private Caster $caster,
+        private string $dateTimeFormat = \DateTimeInterface::ATOM,
```

Comment:
Is this the format that googl sheets recognize automatically as timestamp?

Reply:
This can be handled if we allow passing as user entered for input value in update, if we go with raw we should manipulate the data.